addons/html_editor/static/tests/editor/unbreakable/delete.test.js

```diff
@@ -468,72 +467,66 @@ describe("forward", () => {
-                });
-            }
-        );
+        test("should delete first character of unbreakable, ignoring selected paragraph break (forward)", async () => {
+            await testEditor({
+                contentBefore: '<p>abc[</p><p class="oe_unbreakable">d]ef</p>',
+                stepFunction: deleteForward,
+                contentAfter: '<p>abc[]</p><p class="oe_unbreakable">ef</p>',
+            });
+        });
     });
 });
 
 describe("list", () => {
     describe("selection collapsed", () => {
-        test.todo(
-            "should not outdent while nested within a list item if the list is unbreakable",
-            async () => {
-                // Only one LI.
-                await testEditor({
-                    contentBefore: "<p>abc</p><ol><li>[]def</li></ol>",
-                    stepFunction: deleteBackward,
-                    contentAfter: "<p>abc</p><ol><li>[]def</li></ol>",
-                });
-                // First LI.
-                // await testEditor({
-                //     contentBefore:
-                //         '<ol><li><div><div>[]abc</div></div></li><li>def</li></ol>',
-                //     stepFunction: deleteBackward,
-                //     contentAfter:
-                //         '<ol><li><div><div>[]abc</div></div></li><li>def</li></ol>',
-                // });
-                // // In the middle.
-                // await testEditor({
-                //     contentBefore:
-                //         '<ol><li><div>abc</div></li><li><div><div>[]def</div></div></li><li>ghi</li></ol>',
-                //     stepFunction: deleteBackward,
-                //     contentAfter:
-                //         '<ol><li><div>abc</div></li><li><div><div>[]def</div></div></li><li>ghi</li></ol>',
-                // });
-                // // Last LI.
-                // await testEditor({
-                //     contentBefore:
-                //         '<ol><li>abc</li><li><div><div>[]def</div></div></li></ol>',
-                //     stepFunction: deleteBackward,
-                //     contentAfter:
-                //         '<ol><li>abc</li><li><div><div>[]def</div></div></li></ol>',
-                // });
-                // // With a div before the list:
-                // await testEditor({
-                //     contentBefore:
-                //         '<div>abc</div><ol><li>def</li><li><div><div>[]ghi</div></div></li><li>jkl</li></ol>',
-                //     stepFunction: deleteBackward,
-                //     contentAfter:
-                //         '<div>abc</div><ol><li>def</li><li><div><div>[]ghi</div></div></li><li>jkl</li></ol>',
-                // });
-            }
-        );
+        test("should not outdent while nested within a list item if the list is unbreakable", async () => {
+            // Only one LI.
+            await testEditor({
+                contentBefore: '<p>abc</p><ol t="1"><li>[]def</li></ol>',
```

Comment:
what is t=1 ?

Reply:
No idea... I guess the idea was to trigger this condition in `isUnbreakable`: `node.hasAttribute("t") `
Anyway, the original test had this, and somehow it disappeared when Nicolas moved stuff. See https://github.com/odoo-dev/odoo/commit/cbfa43786ec8669ef93e98e4e2178e43beb09833#r140126285

Reply:
ok